src/components/SearchResults/SearchResults.js

```diff
@@ -15,7 +15,7 @@ export class SearchResults extends React.Component{
         } = this.props;

```

Comment:
this component should not be a class, since it has no methods, it just render html, so pls create a function component